packages/kbn-optimizer/limits.yml

```diff
@@ -25,7 +25,7 @@ pageLoadAssetSize:
   core: 435325
   crossClusterReplication: 65408
   customIntegrations: 22034
-  dashboard: 82025
+  dashboard: 136103
```

Comment:
This is a pretty significant size increase, did we look into making this async?

Reply:
@angorayc - this is a major increase. Do you think that this is completely due to exporting the toolbar? If so we can definitely make it a lazy component.